mlflow/server/js/src/common/components/IconButton.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+
+export class IconButton extends Component {
+  static propTypes = {
+    children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
+    style: PropTypes.object,
+    restProps: PropTypes.object,
```

Comment:
What do you think about adding an optional `type` prop, if present, automatically add children `<Icon type={props.type}` />`

Reply:
What about creating a `icon` prop that takes a JSX.Element?

```js
// antd icon

<IconButton
  ...
  icon={<Icon type="edit">}
/>

// fontawesome icon

<IconButton
  ...
  icon={<i className='far fa-trash-alt' />}
/>
```

Reply:
LGTM!